README.md

```diff
@@ -113,10 +115,33 @@ additional metrics!
 
 See the [`docs`](docs) directory for more information on the exposed metrics.
 
+#### Conflict resolution in label names
+
+`*_labels` family of metrics exposes Kubernetes labels as Prometheus labels.
+[Kubernetes](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set)
+is more liberal than
+[Prometheus](https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels)
+in terms of allowed characters in label names,
+so we automatically convert unsupported characters to underscores,
+e.g. `app.kubernetes.io/name` becomes `label_app_kubernetes_io_name`.
+
+This conversion can create conflicts when multiple Kubernetes labels like
+`foo-bar` and `foo_bar` would be converted to the same Prometheus label `label_foo_bar`.
+
+In order to resolve this conflict, a suffix `_conflictN` is automatically added
```

Comment:
We try to avoid passive voice as much as we can.
```suggestion
Kube-state-metrics automatically adds a suffix `_conflictN` to resolve this conflict,
```